Jobs/PrGate.yml

```diff
@@ -68,64 +64,99 @@ parameters:
   displayName: Virtual Machine Image (e.g. windows-latest)
   type: string
   default: 'windows-latest'
+- name: extra_install_step
+  displayName: Extra Install Steps
+  type: stepList
+  default: []
+- name: pool_name
+  displayName: Variable name that hosts pool name to be used for self-hosted agents
+  type: string
+  default: ''
+- name: os_type
+  displayName: OS type on the self-hosted agent pools
+  type: string
+  values:
+  - Windows_NT
+  - Linux
+  default: Windows_NT
+- name: build_matrix

```

Comment:
I think this comment (or somewhere in this repo's documentation) should give a brief example of a matrix.

Reply:
Agreed, will update.

Reply:
Added two examples of matrix entries in the comment, one that uses self host agent, one that does not.